doc/nrf/libraries/caf/buttons_pm_keep_alive.rst

```diff
@@ -0,0 +1,26 @@
+.. _caf_buttons_pm_keep_alive:
+
+CAF: Buttons power manager keep alive module
+############################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The |buttons_pm_keep_alive| of the :ref:`lib_caf` (CAF) is responsible for generating :c:struct:`keep_alive_event` events on any configured button press.
+
+Configuration
+*************
+
+To use the module, you must enable the following Kconfig options:
+
+* :kconfig:`CAF_BUTTONS_PM_KEEP_ALIVE` - This option enables |buttons_pm_keep_alive|.
+* :kconfig:`CONFIG_CAF_BUTTONS` - This option enables the buttons module.
+* :kconfig:`CAF_KEEP_ALIVE_EVENTS` - This option enables keep alive events.
+
+Implementation details
+**********************
+
+The module reacts on :c:struct:`button_event` and generates :c:struct:`keep_alive_event` if any configured button is pressed.
```

Comment:
Would it make sense to describe here briefly the relation of the module to the power manager module? Or maybe what can happen with the keep_alive_event once it is generated?

Reply:
I think it does not. User should look into keep_alive_event description for that. In other case we would have a kind of the documentation for keep_alive_event in different places in the repository. If anything updates or changes nobody would remember to change it here.

Reply:
OK, and what about just mentioning the relation to power manager? After all, the module has power manager in its name and requires it to function. See my suggestion to the first line.

Reply:
It seems it was introduced by yours fix?